File: Daily Challanges/2026-05-May/2026-05-20.js
function zipStrings(a, b) {
  let result = [];
  let min_len = Math.min(a.length, b.length);
  for (let i = 0; i < min_len; i++) {
    result.push(a[i]);
    result.push(b[i]);
  }
  result.push(a.slice(min_len));
  result.push(b.slice(min_len));
  return result.join("");
}

console.log(zipStrings("abc", "123"));
console.log(zipStrings("day", "night"));

File: Daily Challanges/2026-05-May/2026-05-20.py
def zip_strings(a, b):
    result = []
    min_len = min(len(a), len(b))
    for i in range(min_len):
        result.append(a[i])
        result.append(b[i])
    result.append(a[min_len:])
    result.append(b[min_len:])
    return ''.join(result)

print(zip_strings("abc", "123"))
print(zip_strings("day", "night"))